client/src/legacy/VersionedEditForm/VersionedEditForm.js

```diff
@@ -0,0 +1,55 @@
+import jQuery from 'jquery';
+
+jQuery.entwine('ss', ($) => {
+  /**
+   * Class: .cms-edit-form .btn-toolbar #Form_ItemEditForm_action_doUnpublish
+   * Informing the user about the unpublish action while requiring confirmation
+   */
+  $('.cms-edit-form .btn-toolbar #Form_ItemEditForm_action_doUnpublish').entwine({
+
+    /**
+     * Function: onclick
+     *
+     * Parameters:
+     *  (Event) e
+     */
+    onclick(e) {
+      const message = 'Are you sure you want to remove your record from the published site?\n\nThis record will still be available in the CMS as draft.';
```

Comment:
Ideally we can localise this, i.e. `i18n._t('VersionedEditForm.CONFIRM_MESSAGE', 'Are you sure...')`